Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -224,14 +224,11 @@ private bool canReadPartial(string resource)
                             return true;
                         return false;
                     }
-                // No one except super admin should be able to see all the membership requests a student has
+                // No one except super admin should be able to see all the membership requests a student has.
```

Comment:
Please add blank line above here, to separate cases.

I think this line can be deleted.  It's not really an issue here.  If the super admin needs that access, they must get it somewhere else.  (I think they use APIs that access requests by involvement, not by user.)  Is this right?